spec/twitter-ads/campaign/line_item_spec.rb

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2015 Twitter, Inc.
+
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.